src/website/views/custom-elements/badge.pug

```diff
@@ -0,0 +1,195 @@
+---
+title: Badge
+scripts:
+  - js/ce/ux-chi-ce
+
+---
+p.-text Badges are inline elements used to label, categorize, or organize a list of items.
+
+h3 Examples
+p.-text
+  | To display a badge, use the tag <code>&lt;chi-badge&gt;</code>.
+  | A contextual color (e.g. <code>dark</code>) can also be set to <code>color</code> attribute to provide alternate styling.
+
+h4 Base
+.example.-mb--3
+  .a-grid.-noGutter
+    .a-col.-w--12
+      .-p--2.-text--center
+        chi-badge(class='-m--1')= 'Base'
```

Comment:
Why not adding the -m--1 class to the implementation of the component instead of including it always as a parameter?

Reply:
We don't want utility classes to be part of CE. I'm using it everywhere just to make docs look prettier that's all.